schema.sql

```diff
@@ -6,5 +6,24 @@ CREATE TABLE animals (
     date_of_birth date not null,
     escape_attempts int not null,
     neutered boolean not null,
-    weight_kg decimal not null
+    weight_kg decimal not null,
+    species varchar(100)
 );
```

Comment:
- [ ] As demanded by project requirements, kindly remove the species column from the animals' table. You can use `ALTER TABLE  REMOVE species;` command.

Reply:
![Screenshot 2023-05-04 231418](https://user-images.githubusercontent.com/109290913/236349487-51a67fd5-07d0-442f-8e49-80ec08428ebc.png)

Reply:
Needs attention!
Screenshots should be updated within the PR description.